utils/lib_sxng_test.sh

```diff
@@ -100,11 +101,17 @@ test.types.ci() {
 }
 
 test.black() {
-    build_msg TEST "[black] \$BLACK_TARGETS"
+    build_msg TEST "[black] ${BLACK_TARGETS[*]}"
```

Comment:
The dollar sign was deliberately escaped  here, output should be:

    TEST      [black] $BLACK_TARGETS

.. not the value of the environment variable.